tables/SV-SE-G1.utb

```diff
@@ -0,0 +1,506 @@
+# liblouis: Swedish Grade 1 Braille Table
+#
+#  Copyright (C) 2022 Insyn Scandinavia AB <david@insyn.se>
+#  Copyright (C) 2012 Mesar Hameed <mesar.hameed@gmail.com>
+#  Copyright (C) 2004-2008 ViewPlus Technologies, Inc. www.viewplus.com
+#  Copyright (C) 2004-2006 JJB Software, Inc. www.jjb-software.com
+#
+#  This file is part of liblouis.
+#
+#  liblouis is free software: you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as
+#  published by the Free Software Foundation, either version 2.1 of the
+#  License, or (at your option) any later version.
+#
+#  liblouis is distributed in the hope that it will be useful, but
+#  WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with liblouis. If not, see
+#  <http://www.gnu.org/licenses/>.
+
+# Characters not to backtranslate (must be included for braille input to work)
+include braille-patterns.cti
+
+# Spaces
+space \x00a0 0
+include spaces.uti
+
+# Punctuation
+punctuation ! 235			exclamation mark		x0021

```

Comment:
I know most tables include comments like this one ("exclamation mark") after character definitions, but actually such comments are mostly redundant and just clutter up the table. We're thinking of doing a big cleanup of unneeded comments in all existing tables. For new tables however maybe it is better to not include the comments in the first place.